reframe/core/containers.py

```diff
@@ -88,9 +88,6 @@ def validate(self):
         if self.image is None:
             raise ContainerError('no image specified')
 
-        if not self.commands:
```

Comment:
The creator of an images should know best how to execute the command. E.g. for Spack the `ENTRYPOINT` needs to be honored.
I'd like to argue that a container should also include the execution script.